### DISCUSSION TITLE
tooltip custom parent

### DISCUSSION THREAD

**Discussion by steviesocks** [heroui-inc/heroui#2054] [Q&A]
Is it possible to set a custom parent element for Tooltip? Tooltips in "click" trigger mode only scroll with the target if the `<body>` element scrolls (since `<div id="nextui-tooltip">` is a child of `<body>`). This leaves the tooltip floating, detached from its target if scrolling happens inside a different container. Thanks!

**Comment by Tolc** [comment]
Would also like to know this.

Nothing to do with scroll in my case though, I need it for a complex layout.
